docs/experience/integrations.md

```diff
@@ -0,0 +1,207 @@
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+	<div class="mdl-cell mdl-cell--4-col">
+		<a href="#vuejs">
+			<div class="demo-card-event mdl-card mdl-shadow--2dp">
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../images/integrations/vuejs.png" alt="Vue.js" title="Vue.js" style="padding-top:40px;height:110px;">
+						</div>
+						<div class="side back">
+							<h5>Vue.js</h5>Full support for single-page applications built with Vue.js.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+	<div class="mdl-cell mdl-cell--4-col">
+		<a href="#ember">
+			<div class="demo-card-event mdl-card mdl-shadow--2dp">
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../images/integrations/ember.png" alt="Ember" title="Ember" style="padding-top:40px;height:110px;">
+						</div>
+						<div class="side back">
+							<h5>Ember</h5>Full support for single-page applications built with the Ember framework.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+  <div class="mdl-cell mdl-cell--4-col">
+		<a href="#static-websites">
+			<div class="demo-card-event mdl-card mdl-shadow--2dp">
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../images/integrations/static.png" alt="Static Websites" title="Static Websites" style="padding-top:40px;height:110px;">
+						</div>
+						<div class="side back">
+							<h5>Static Websites</h5>Full support for all types of static websites.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+  <div class="mdl-cell mdl-cell--4-col">
+		<a href="#server-side-rendered-websites">
+			<div class="demo-card-event mdl-card mdl-shadow--2dp">
+				<div class="flip-card-container">
+					<div class="flip-card">
+						<div class="side">
+							<img src="../images/integrations/server.png" alt="Server-side rendered websites" title="Server-side rendered websites" style="padding-top:40px;height:110px;">
+						</div>
+						<div class="side back">
+							<h5>Server-side rendered websites</h5>Full support for all types of SSR websites and applications.
+						</div>
+					</div>
+				</div>
+			</div>
+		</a>
+	</div>
+</div>
+
+All of the integrations require adding and [configuring the Experience script](./getting-started). However, single-page applications require one more configuration step to register route changes.
+
+## Angular
+The Angular SPA integrations supports both Angular.js and Angular 2+. 
+
+### Angular 2+
+If your website uses newer Angular.js you should add a call to routeChange whenever the route changes in your webapp. This can be done in your top-level component where Router is defined by adding the ngOnInit function and subscribing to the Router events as shown in the following example.
```

Comment:
website or webapp or app or frontend app or ...?

routeChange => `routeChange` ?
Router => `Router`?

Reply:
Webapp.
Fixed.

Reply:
ngOnInit => `ngOnInit` ?

Reply:
Yep, I added that fix as part of the last commit as well. :sunglasses: